src/trackers/UNIT3D/DP.py

```
@@ -11,29 +11,84 @@
+        "ToVaR",
+        "Trix",
+        "TSP",
+        "TSPxL",
+        "ViSION",
+        "VXT",
+        "WAF",
+        "WKS",
+        "X0r",
+        "YIFY",
+        "YTS",
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)